resources/builder.inc.sh

```diff
@@ -1389,6 +1391,12 @@ _builder_parse_expanded_parameters() {
           _builder_chosen_options+=(--debug)
           builder_debug=--debug
           ;;
+        --release|-r)
```

Comment:
```suggestion
        --release)
```

We won't be using this much, so the shorthand flag doesn't need to be used up for it. In fact, we can and should get rid of `-d` for debug as well as it is default.